packages/x-components/src/x-modules/search/store/__tests__/actions.spec.ts

```diff
@@ -320,6 +320,106 @@ describe('testing search module actions', () => {
     });
   });
 
+  describe('resetState', () => {
+    it('should not reset the page when the page parameter of the request changes', async () => {
```

Comment:
Might be interesting to add a little comment to highlight this https://github.com/empathyco/x/pull/283#discussion_r774981772 

at first sight I was confused until I realised that we were only resetting some fields. I can see how someone else could face the same issue 😅